src/gui/menus/AboutMenu.cpp

```diff
@@ -14,6 +14,7 @@ void AboutMenu::drawTop() const
 	Gui::Print("About:", 20.0f, 20.0f, 0.5f, 0.5f);
 	Gui::Print("LimePlayer3DS version " LIMEPLAYER_VERSION, 40.0f, 40.0f, 0.5f, 0.5f);
 	Gui::Print("This application is licensed under GPL3.0-or-later.", 40.0f, 60.0f, 0.5f, 0.5f);
+	Gui::Print("Build" BUILD "(Beta)", 40.0f, 50.0f, 0.5f);
```

Comment:
The BUILD macro is not defined.

The commit number is already apart of the LIMEPLAYER_VERSION variable, but it may make sense to put the build date.